docs/specifications/domain-features/document-digital-signing.md

```
@@ -278,14 +279,194 @@ let scheduleRenewalNotification (signature: DigitalSignature) =
         createScheduledNotification signature.SignerInfo.Email message (signature.ExpiresAt.AddDays(-days)))
 ```
 
+## Command Line Interface
+
+### Basic Signature Command
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)